beetsplug/fetchart.py

```diff
@@ -975,6 +974,26 @@ def get(self, album, plugin, paths):
             self._log.debug('Spotify: error loading response: {}'
                             .format(response.text))
             return
+
+
+class CoverArtUrl(RemoteArtSource):
```

Comment:
Maybe a docstring would be nice here to explain the (somewhat nuanced) role this source plays?

Reply:
Added